public/app/plugins/datasource/prometheus/specs/datasource_specs.ts

```diff
@@ -228,6 +228,53 @@ describe('PrometheusDatasource', function() {
       expect(results[0].time).to.be(123 * 1000);
     });
   });
+  describe('When performing annotationQuery with seriesValueAsTiemstamp', function() {
```

Comment:
Please add this test suite to the .jest.ts file. We're migrating away from Karma tests.